glsk/glsk-commons/src/test/resources/TestCase12Nodes.uct

```diff
@@ -0,0 +1,38 @@
+##C 2007.05.01
```

Comment:
Can we transfer this use case on powsybl-core side and maybe create a factory as we do for CGMES use cases?